zkevm-circuits/src/tx_circuit.rs

```diff
@@ -3444,7 +3740,22 @@ impl<F: Field> TxCircuit<F> {
                     }
                 };
 
+                let mut chunk_bytes: Vec<u8> = vec![];
+                for i in 0..sigs.len() {
+                    let tx = get_tx(i);
+                    if (tx.tx_type != TxType::L1Msg) && !tx.caller_address.is_zero() {
+                        chunk_bytes.extend_from_slice(&tx.rlp_signed);
+                    }
+                }
+
+                let chunk_txbytes_hash = keccak256(chunk_bytes.as_slice());
+                let evm_word = challenges.evm_word();
```

Comment:
can we use `challenges.keccak_challenge();` here?

Reply:
No, because we are storing keccak digest's result.